scipy/spatial/transform/rotation.py

```diff
@@ -1463,6 +1464,73 @@ def magnitude(self):
         else:
             return angles
 
+    def reduce(self, left=None, right=None, return_best=False):
+        """Reduce this rotation with the provided rotation groups.
+
+        Parameters
+        ----------
+        left : `Rotation` instance
+            Object containing the left rotation group.
+        right : `Rotation` instance
+            Object containing the right rotation group.
+        right : bool
```

Comment:
return_best, optional

Perhaps return_indices is a better name. But use your best judgement (as always).

Reply:
`return_indices` is more informative.  I have renamed it.